var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -35,36 +34,20 @@
 
 class Jdk(Package):
     """The Java Development Kit (JDK) released by Oracle Corporation
-       in the form of a binary product aimed at Java developers."""
+       in the form of a binary product aimed at Java developers.
+
+      For legal reasons, Spack is not able to auto-download this tarball.
+      To install, do as follows:
+        1. Set up a Spack mirror.
+        2. Manually download the JDK tarball and put it in the Spack mirror.
```

Comment:
> This should mention var/spack/cache as the default location; those using
mirrors probably know that they're doing so.

I disagree.  `/var/spack/cache` is auto-generated and large.  It's the kind
of thing users will "blow away" from time to time.  Manually downloaded
tarballs should go into a more carefully curated Spack mirror that is
maintained, backed up, etc.



On Tue, Jul 19, 2016 at 12:02 PM, Ben Boeckel <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/jdk/package.py:
>
> > @@ -35,36 +34,20 @@
> >
> >  class Jdk(Package):
> >      """The Java Development Kit (JDK) released by Oracle Corporation
> > -       in the form of a binary product aimed at Java developers."""
> > +       in the form of a binary product aimed at Java developers.
> > +
> > +      For legal reasons, Spack is not able to auto-download this tarball.
> > +      To install, do as follows:
> > +        1. Set up a Spack mirror.
> > +        2. Manually download the JDK tarball and put it in the Spack mirror.
>
> This should mention var/spack/cache as the default location; those using
> mirrors probably know that they're doing so.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1298/files/cdd0ca70c4a706f35901b883c8183e40773aa0bf#r71369037>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cd3J9seXCGjVnEVUJGdnj_osNalBEks5qXPUFgaJpZM4JPx0z>
> .
>